Kickstarter-iOS/Views/Controllers/DashboardViewController.swift

```diff
@@ -23,6 +25,10 @@ internal final class DashboardViewController: UITableViewController {
   internal override func viewDidLoad() {
     super.viewDidLoad()
 
+    self.tableView.backgroundView = self.backgroundView
```

Comment:
This approach feels like we're fighting `UIKit`. I know `UITableViewController`s can be pretty stubborn when it comes to subviews, but I'm wondering if it wouldn't be better to do this using an empty state or a cell type that we can display while loading.